modules/build/src/main/scala/scala/build/preprocessing/ScriptPreprocessor.scala

```diff
@@ -67,6 +68,10 @@ object ScriptPreprocessor {
 
     val (requirements, scopedRequirements, options, updatedCodeOpt) =
       value(ScalaPreprocessor.process(contentIgnoredSheBangLines, reportingPath, scopePath / os.up))
+        .map {
+          case ProcessingOutput(globalReqs, scopedReqs, opts, updatedContent) =>
```

Comment:
Maybe we should keep the `ProcessingOutput` here (and not match above). That case only "unapplies" `ProcessingOutput`.